ruby/lib/google/protobuf.rb

```diff
@@ -35,6 +35,98 @@
 # That way the module init can grab references to these.
 module Google
   module Protobuf
+    # A pointer -> Ruby Object cache that keeps references to Ruby wrapper
```

Comment:
Can we move these class definitions to their own file?

Reply:
Done!